xtask/src/transfer_workload.rs

```diff
@@ -0,0 +1,54 @@
+use crate::{
+    backend::{Action, DB},
+    timer::Timer,
+};
+
+pub fn transfer_workload(
```

Comment:
Let's adjust this slightly.

There are broadly three things we care about:
  1. The size of the account database
  2. The amount of transfers to perform
  3. The proportion of the database to use for transfers - some state will be warm, some will be cold

You can come up with sub-categories for each of these, but I think we should just change the function for now to accept a parameter for each one of these.

Reply:
Interesting metrics! Initially, I envisioned it differently when trying to find a good common divisor for the custom workload. I am not sure how to handle the percentage of warm/cold storage without making the CLI messy.

I will attempt to further decouple the transfer workload from the custom workload, which may make it easier to develop new workloads and address those three points!